Tutorial HTML/Html 01/README.md

```diff
@@ -0,0 +1,85 @@
+# Aplicatii web pentru incepatori
+######  Autor : Oancea Ionut Eugen
+
+![HACKED.io: Hands on FirefoxOS](https://arcturus.github.io/firefoxos-hackedio/img/html5_css_javascript.png)
+
+Orice aplicatie web acesibila intr-un browser foloseste o combinatie de 3 limbaje pentru a ajunge la forma sa finala (HTML + CSS + JavaScript). 
+Pentru a continua ne este indeajuns sa intelegem pe scurt rolul fiecarui limbaj intr-o aplicatie
+Nu ne vom arunca prea mult inca in detalii la acest punct.
+
+- *Am adaugat un link catre wikipedia cu niste explicatii, pentru cei care doresc insa acest lucru.*
+
+#### - HTML ( Hyper text markup language )
+[HTML Wikipedia](https://ro.wikipedia.org/wiki/HyperText_Markup_Language)
+Este folosit pentru a crea pagini web simple ce pot fi afișate într-un browser.
```

Comment:
Never Wikipedia, sa folosim MDN ca resursa principala cand dam linkuri spre documentatii, explicatii, etc.

Reply:
Eh, ok, nu am considerat in prima faza sursa atat de critica, dar am sa tin cont de asta, pentru ca pana la urma ei au si exemple si explicatii

Reply:
Da, hai sa-i invatam cu MDN si alte surse semi-oficiale cum ar fi [web.dev](https://web.dev) sau [Google Chrome devs](https://developers.google.com/web/updates). Din experienta de predat, conteaza mult sa nu fie W3Schools sau alte prostii :)))